### PR TITLE
Bring a random tour tip into the front page

### PR DESCRIPTION
**Pull request by laurieboyes**
screen shot 2016-09-12 at 17 29 17

Depends on https://github.com/Financial-Times/n-ui/pull/419

Behind the `nextFtTour` flag

### CODE REVIEW

File: client/components/tour-tip/_tour-tip.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+				color: oColorsGetPaletteColor('grey-tint4');
+			}
+		}
+	}
+
+	.section__column__inner:before {
+		background-color: $tour-tip-background-color;
+	}
+
+	#tour-tip-section-content {
```
Comment (debugwand) [inline client/components/tour-tip/_tour-tip.scss]: ID selector really necessary?

File: client/components/tour-tip/_tour-tip.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	.section-meta__title {
+		background-color: $tour-tip-background-color;
+		margin-right: 0 !important;
+		top: 3px;
+
+		a {
+			border-bottom: 0;
+			color: oColorsGetPaletteColor('grey-tint5');
+
+			&:hover {
```
Comment (debugwand) [inline client/components/tour-tip/_tour-tip.scss]: is there any feedback other than color change that informs the user that this is a link?

File: client/components/tour-tip/_tour-tip.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+		margin-top: 4px;
+
+		@include oGridRespondTo(S) {
+			line-height: 40px;
+		}
+	}
+
+	.section-meta__title {
+		background-color: $tour-tip-background-color;
+		margin-right: 0 !important;
```
Comment (debugwand) [inline client/components/tour-tip/_tour-tip.scss]: why is !important required?

File: config/pages.js [hunk 4]
```diff
@@ -102,6 +102,11 @@ const getPages = (flags, region, lifeAndArtsHigh) => {
 	// Insert Life & Arts component into position dictated by time of week
 	layout.splice(layout.indexOf(lifeAndArtsHigh ? 'opinion' : 'video'), 0, 'life-and-arts');
 
+	const afterOpinion = layout.indexOf('opinion') + 1;
```
Comment (debugwand) [inline config/pages.js]: can you drop this into the following if statement? it won't be in a temporal dead zone then

**Comment by debugwand** [comment]
a few comments, otherwise good

File: client/components/tour-tip/_tour-tip.scss (hunk 2 above)
Comment (laurieboyes) [inline client/components/tour-tip/_tour-tip.scss]: Yeah hidden link is bad, will consult a designer

Comment (laurieboyes) [inline client/components/tour-tip/_tour-tip.scss]: Seems to be part of a wider issue of not-very-obvious links on the homepage. I think there's some design rethought happening on the homepage currently so I'll chat with the people behind that and make sure they're conscious of this, and consider it non-blocking for this PR.

**Pull request merged**